### ISSUE TITLE
Reduce zombie awareness values

### ISSUE THREAD

**Issue by Forecaster** [Forecaster/RailcraftPatronServerIssues#51]

**Comment by liach** [comment]
What values?

**Comment by CovertJaguar** [comment]
Yes, I've looked at them, I don't think they do what you want them to do. Which I assume is reduce the number of mobs.

**Comment by Forecaster** [comment]
I was going to reduce the max amounts in `ZombieAwareness/Spawning.cfg`

Specifically `I:extraSpawningSurfaceMaxCount=50`
